lib/injection.rb

```diff
@@ -0,0 +1,10 @@
+class Array
+
+  def injection(num = 0, &block)
```

Comment:
Unused method argument - `block`. If it's necessary, use `_` or `_block` as an argument name to indicate that it won't be used.